### DISCUSSION TITLE
Getting error when trying to merge my PR

### DISCUSSION THREAD

**Discussion by PaulieB14** [vercel/community#1101] [Help] [answered]
Here is link and inside contains error - any suggestions - https://github.com/yearn/yearn-devdocs/pull/281?notification_referrer_id=NT_kwDOBaXOvbM0OTM3OTg5NDUxOjk0NzUyNDQ1#issuecomment-1329906212
![204403093-fb36c845-ac08-4c28-a7c6-f786c833e3ea](https://user-images.githubusercontent.com/94752445/204412286-8f03c697-8090-4f85-b5eb-10ad56724ff7.png)

**Comment by swarnava** [comment, marked as answer]
This looks like an error in your code base, you should be able to reproduce this if you run `build` command.

**Reply by PaulieB14** [reply]
Thank you.  I think it was using <br>, instead of <br /> which left unclosed tag.
